lib/job-iteration/csv_enumerator.rb

```diff
@@ -30,19 +30,23 @@ def initialize(csv)
     # Constructs a enumerator on CSV rows
     # @return [Enumerator] Enumerator instance
     def rows(cursor:)
```

Comment:
Is it worth documenting what `cursor` means for the iteration on resume (specifically that it represents the last _successfully_ processed row?)

Reply:
Yeah it would make sense